eng/common/scripts/Test-SampleMetadata.ps1

```diff
@@ -269,6 +269,7 @@ begin {
         "azure-search",
         "azure-security-center",
         "azure-sentinel",
+        "azure-service-connector",
```

Comment:
Is that the correct one or should it be azure-service-linker?

Reply:
I don't see either at https://review.docs.microsoft.com/en-us/help/contribute/metadata-taxonomies?branch=main#product. can you please find one that is on that list? Also can you update the link in the comments to that same link (i.e. replace master with main).

Reply:
I don't know how to add to the list. Is it any docs about it?
The product name is Service Connector as I mentioned above, I'm not sure it should be azure-service-linker or azure-service-connector. What is the guidance?

Reply:
I'm less familiar with these doc slugs. @heaths @sima-zhu @danieljurek any recommendations about these service slugs?

Reply:
Thanks wes, maybe I'll add "azure" only in product for sample if this metadata is hard to add

Reply:
I don't have a lot of context so I cannot provide good guidance here but I think you could try using "azure" as a starting point. 

@scottaddie do you have any suggestions on this front?

Reply:
It is usually in a format of "word-word". 
Found `service-connector` in taxonomies. 
Check [here](https://taxonomy.docs.microsoft.com/TaxonomyServiceAdminPage/#/taxonomy/detail/2022-04-27T01:35:49.5155761Z!669b6b65-c764-4717-bbd3-77d5c8065769/ms.service)

Reply:
`service-connector` is also my recommendation